README.md

```diff
@@ -9,3 +9,29 @@ names and labels by using a set of regular expressions.
 
 Please refer to this utility's `main()` function for a list of supported
```

Comment:
I think this sentence could be removed/adjusted if we're going to document the command line flags below, right?